target/linux/ramips/mt7621/base-files/lib/preinit/04_handle_checksumming

```diff
@@ -0,0 +1,27 @@
+do_checksumming_disable() {
+	case "$(board_name)" in
+	asus,rp-ac56)
+		local firmware_mtd=$(find_mtd_part firmware)
+		local rootfs_mtd=$(find_mtd_part rootfs)
+		local rootfs_data_mtd=$(find_mtd_part rootfs_data)
+		local rootfs_len=$(grep \"rootfs\" /proc/mtd | awk -F' ' '{print "0x"$2}')
+		local rootfs_data_len=$(grep \"rootfs_data\" /proc/mtd | awk -F' ' '{print "0x"$2}')
+		local offset=$(echo "$rootfs_len $rootfs_data_len 0x40" | awk -F' ' '{printf "%i",$1-$2-$3}')
+		local signature=$(dd if=$rootfs_mtd skip=$offset bs=1 count=4 2>/dev/null | hexdump -v -n 4 -e '1/1 "%02x"')
```

Comment:
dd if=$rootfs_mtd skip=$offset bs=4 count=1 iflag=skip_bytes
Despite, can this be done with hexdump directly?

Reply:
Done, using hexdump directly, added that optimization, although I'd prefer readability vs performance here